projects/sensory_interface/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<title> Sensory interface </title>
+
+<head>
+    <script src="app.js"></script>
+</head>
+
+<body>
+    <textarea id="textInput">1	2	3
+4	5	6</textarea>
+    <button id=" processButton" onclick="processData()"> process </button>
```

Comment:
You have a space at the beginning of `processButton` that you probably want to remove.